lib/trino-memory-context/src/main/java/io/trino/memory/context/ThresholdLocalMemoryContext.java

```diff
@@ -1,93 +0,0 @@
-/*
```

Comment:
nit: this should be two commits:
1. rename
2. changes on top
this way history is preserved